shared/chat/conversation/messages/index.js

```diff
@@ -13,8 +13,12 @@ import {Box} from '../../../common-adapters'
 const factory = (
   messageKey: Constants.MessageKey,
   prevMessageKey: ?Constants.MessageKey,
-  onAction: (message: Constants.ServerMessage, event: any) => void,
-  onShowEditor: (message: Constants.ServerMessage, event: any) => void,
+  onAction: (
```

Comment:
Fixed arguments here to match passed-in ones (I'm surprised flow didn't catch this!)